todo_app/data/trello_items.py

```diff
@@ -0,0 +1,59 @@
+import requests
+import os
+
+from todo_app.data.item import Item
+
+def add_item(title):
+
+    reqUrl = "https://trello-proxy.azure-api.net/1/cards"
+
+    query_params = {
+        "key": os.getenv("TRELLO_API_KEY"),
+        "token": os.getenv("TRELLO_API_TOKEN"),
+        "idList": os.getenv("TODO_LIST_ID"),
+        "name": title
+    }
+
+    response = requests.post(reqUrl, params = query_params)
+
+    print(response.text)
```

Comment:
Again minor, but we should always think for a moment about logging that we've left in the app - could it leak personal data? Here for example, if we look at what this prints it looks like it includes lots of detail about the card created, which is really helpful for debugging but definitely has some risks if this hit production; I can certainly imagine people storing personal data in their to-dos like:
* Visit doctor about <condition>
* Email <address> about <thing>
* Buy 50th birthday present for <name>

Typically we want to keep all of that sort of info out of our logging, so that we can control who has access to that data without limiting who can see the logs